src/libYARP_dev/src/devices/FrameTransformServer/FrameTransformServer.cpp

```diff
@@ -36,7 +36,7 @@ yarp::dev::DriverCreator *createFrameTransformServer() {
 bool Transforms_server_storage::delete_transform(int id)
 {
     LockGuard lock(m_mutex);
-    if ((size_t)id >= 0 && (size_t)id < m_transforms.size())
+    if ((size_t)id < m_transforms.size())
```

Comment:
What will happen if the input `id` is negative? I think that check still needs to be done, maybe in a different way.